### DISCUSSION TITLE
PBPK-PD model

### DISCUSSION THREAD

**Discussion by MaLuisaEti** [Open-Systems-Pharmacology/Forum#1486] [Models]
Hi! 
I am working on a PBPK model and I´d like to evaluate my drug´s effect on urinary sodium excretion.
I have the following data: plasma concentrations of my drug over time, urinary excretion of my drug over time, and urinary sodium excretion over time.
What strategy would you recommend? May be, the easiest way is to find an equation that relates urinary sodium excretion to drug´s  plasma concentration or drug´s urinary excretion and add it as a parameter in Mobi. But I wonder if from a mechanistic point of view it is a better way.

If anyone can help me, I will appreciate it very much!

Thank you in advance
Ma Luisa

**Comment by StephanSchaller** [comment]
Hi @MaLuisaEti ,

It depends on how much detail you want to put into the model.
We have published a kidney model which could be adopted and extended with relevant PD mechanisms to model sodium excretion.: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6310870/
It could also be further extended with the mechanisms published by M Hallow et al.

But a simple solution as described by you might be a sufficient first step.

Best,
Stephan

**Comment by PavelBal** [comment]
Hi @MaLuisaEti , you can find the model that @StephanSchaller refers to here: https://github.com/Open-Systems-Pharmacology/SGLT2i-hyperfiltration-model

**Comment by MaLuisaEti** [comment]
@StephanSchaller and @PavelBal thank you very much for your answers!
A week ago I had found that work but when I wanted to open the .mbp3 file of the supplementary material it gave me an error message (I assumed it was because of the MoBi version). But now I was able to access it without problems through the download link on GitHub.

I'll be looking at your kidney model! 

Thanks again!
